src/components/graph-component/util/get-value.util.js

```diff
@@ -0,0 +1,35 @@
+import { ONE_HUNDREED } from "../../../core/index";
+
```

Comment:
if you return { error } in diff functions, i suppose you should clarify it more properly